### ISSUE TITLE
InternalExecutionError

### ISSUE THREAD

**Issue by svrc-pivotal** [cloudfoundry-incubator/bosh-azure-cpi-release#86]
```
  Started updating job etcd_server-partition-e4fffe2eeec37a9cd821 > etcd_server-partition-e4fffe2eeec37a9cd821/0 (canary)
  Started updating job nats-partition-e4fffe2eeec37a9cd821 > nats-partition-e4fffe2eeec37a9cd821/0 (canary)
  Started updating job nfs_server-partition-e4fffe2eeec37a9cd821 > nfs_server-partition-e4fffe2eeec37a9cd821/0 (canary)
  Started updating job mysql_proxy-partition-e4fffe2eeec37a9cd821 > mysql_proxy-partition-e4fffe2eeec37a9cd821/0 (canary)
  Started updating job consul_server-partition-e4fffe2eeec37a9cd821 > consul_server-partition-e4fffe2eeec37a9cd821/0 (canary)
  Started updating job mysql-partition-e4fffe2eeec37a9cd821 > mysql-partition-e4fffe2eeec37a9cd821/0 (canary)
  Started updating job router-partition-e4fffe2eeec37a9cd821 > router-partition-e4fffe2eeec37a9cd821/0 (canary)
  Started updating job diego_database-partition-e4fffe2eeec37a9cd821 > diego_database-partition-e4fffe2eeec37a9cd821/0 (canary)
   Failed updating job mysql-partition-e4fffe2eeec37a9cd821 > mysql-partition-e4fffe2eeec37a9cd821/0 (canary): Unknown CPI error 'Bosh::AzureCloud::AzureError' with message 'status: Failed
http code: 200
request id: 78e635f6-1179-4e0f-a440-daf8c93c3a88
error:
{"code"=>"InternalExecutionError", "message"=>"An internal execution error occurred."}' (00:01:11)^C
```

This error commonly occurs when updating jobs that require re-attaching a persistent disk.  I've seen it for several hours now (westus , Standard_D's and D_V2's). 

I've found that deleting the VM manually and deleting the VM reference via `bosh cck` helps, but I have to rerun bosh deploy a lot before they all re-attach.

**Comment by AbelHu** [comment]
Thanks for reporting it. We have started to investigate this issue. Will update you after we find the root cause.

Best Regards,
Abel HU

? 2016?2?1??18:31?Stu Charlton <notifications@github.com<mailto:notifications@github.com>> ???


  Started updating job etcd_server-partition-e4fffe2eeec37a9cd821 > etcd_server-partition-e4fffe2eeec37a9cd821/0 (canary)
  Started updating job nats-partition-e4fffe2eeec37a9cd821 > nats-partition-e4fffe2eeec37a9cd821/0 (canary)
  Started updating job nfs_server-partition-e4fffe2eeec37a9cd821 > nfs_server-partition-e4fffe2eeec37a9cd821/0 (canary)
  Started updating job mysql_proxy-partition-e4fffe2eeec37a9cd821 > mysql_proxy-partition-e4fffe2eeec37a9cd821/0 (canary)
  Started updating job consul_server-partition-e4fffe2eeec37a9cd821 > consul_server-partition-e4fffe2eeec37a9cd821/0 (canary)
  Started updating job mysql-partition-e4fffe2eeec37a9cd821 > mysql-partition-e4fffe2eeec37a9cd821/0 (canary)
  Started updating job router-partition-e4fffe2eeec37a9cd821 > router-partition-e4fffe2eeec37a9cd821/0 (canary)
  Started updating job diego_database-partition-e4fffe2eeec37a9cd821 > diego_database-partition-e4fffe2eeec37a9cd821/0 (canary)
   Failed updating job mysql-partition-e4fffe2eeec37a9cd821 > mysql-partition-e4fffe2eeec37a9cd821/0 (canary): Unknown CPI error 'Bosh::AzureCloud::AzureError' with message 'status: Failed
http code: 200
request id: 78e635f6-1179-4e0f-a440-daf8c93c3a88
error:
{"code"=>"InternalExecutionError", "message"=>"An internal execution error occurred."}' (00:01:11)^C


This error commonly occurs when updating jobs that require re-attaching a persistent disk. I've seen it for several hours now.

I've found that deleting the VM in Azure Portal and/or changing storage accounts for that VM can help, but I have to rerun bosh deploy a lot before they all re-attach.

-
